python/tests/stress/arcticdb/version_store/test_sparse.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import pandas as pd
+
+
+def get_dataframe_dense_till(total_size=100, dense_till=50):
+    arr = []
+    for idx in range(total_size):
+        if idx < dense_till:
+            arr.append(float(idx))
+        else:
+            arr.append(np.nan)
+
+    return pd.DataFrame({"float": arr})
+
+
+def get_interleaved_dataframe(size=100):
+    arr = []
+    for idx in range(size):
+        if idx % 2 == 0:
+            arr.append(float(idx))
+        else:
+            arr.append(np.nan)
+
+    return pd.DataFrame({"float": arr})
+
+
+def test_sparse_interleaved(sym, lmdb_version_store):
+    lib = lmdb_version_store
+    df = get_interleaved_dataframe(100)
+    lib.write(sym, df, dynamic_schema=True)
+    dd = lib.read(sym, dynamic_schema=True).data
+
+    assert dd["float"][2] == df["float"][2]
+
+    # TODO: This should be NaN
+    assert np.isnan(dd["float"][1])
+
+
+def test_sparse_chunked(sym, lmdb_version_store):
+    lib = lmdb_version_store
+    df = get_dataframe_dense_till(100, 30)
+    lib.write(sym, df, allow_sparse=True)
+    dd = lib.read(sym, allow_sparse=True).data
+    assert dd["float"][2] == df["float"][2]
+
+    # TODO: This should be NaN
```

Comment:
What? No they shouldn't?

Assertion on the values that actually should be NaN

Reply:
Removed